test/actions/katello/content_view_puppet_environment_test.rb

```diff
@@ -96,6 +96,23 @@ class CloneToEnvironmentTest < TestBase
     end
   end
 
+  class CopyContentsTest < TestBase
+    let(:action_class) { ::Actions::Pulp::ContentViewPuppetEnvironment::CopyContents }
+    let(:action) { create_action action_class }
+
+    it 'plans and can invoke' do
+      target = puppet_env
+      source = katello_repositories(:p_forge)
+      puppet_module = source.puppet_modules.first
+
+      plan_action action, target, :source_repository_id => source.id, :puppet_modules => [puppet_module ]
```

Comment:
is the space in `[puppet_module ]` on purpose?

Reply:
nope, i'm surprised :policeman: didn't catch this

Reply:
@beav updated